app/views/enroll_course/users.json.jbuilder

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
```

Comment:
As above: is the a duplicated view?

Reply:
Yeah, it is ,and I'll definetly resolve the problems in it